qiita_db/environment_manager.py

```diff
@@ -387,6 +383,10 @@ def patch(patches_dir=PATCHES_DIR, verbose=False):
 
         for sql_patch_fp in sql_patch_files[next_patch_index:]:
             sql_patch_filename = basename(sql_patch_fp)
+
+            if sql_patch_filename == '43.sql' and test:
```

Comment:
Can you add a comment explaining that patch 43 is when we begun testing patches, etc?